dictionaries/software-terms/dict/softwareTerms.txt

```diff
@@ -1434,6 +1460,18 @@ process
 processed
 processes
 procfs
+productisation
+productisations
+productise
```

Comment:
Both `productize` and `productise` are considered valid British English spellings of the same word. But only `productize` is considered valid in US English.

Since `productise` is already in the en-GB, I don't think we need to add it here.